plugins/pulp_python/plugins/models.py

```diff
@@ -8,20 +8,54 @@
 
 
 DEFAULT_CHECKSUM_TYPE = 'sha512'
-REQUIRED_ATTRS = ('name', 'version', 'author', 'author_email', 'description', 'summary')
+REQUIRED_ATTRS = ('name', 'version', 'author', 'summary')
 
 
 class Package(FileContentUnit):
     """
     This class represents a Python package.
+
+    Packages stored on PyPI have significantly more metadata associated with them but we have
+    chosen to keep this model minimal. Below is an explanation for why this was done.
+
+    There are some overloaded terms that should be briefly discussed. We will be using the
+    semantics of PEP 426. https://www.python.org/dev/peps/pep-0426/#id14
+
+    Package - An individual, installable python package. A package is of a specific version, and
+              may also require a specific architecture and python version. Additionally, there
+              may be seperate packages for each package type (sdist, whl, zip).
```

Comment:
Maybe:
```
Package - An individual, installable Python package. A package is defined by the version of
    the project, target architecture, and Python version. Note that there may not be a target
    architecture in the case of architecture-agnostic packages. A Python version is also
    optional if there are no requirements for a specific runtime version. There are three
    packaging formats: sdist, whl, and zip.
```
I'm not convinced the above is better, so pick and choose as you like. If you stick with what you have, ``s/seperate/separate``